spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -18,6 +31,12 @@
     context "on #{plat[:name]} we should behave ourselves" do
```

Comment:
'we should behave ourselves' is redundant.  The context is the platform name.  'Should' statements appear in tests ('it should ....')